tests/header/header_test.cpp

```diff
@@ -45,6 +45,7 @@ class TEST_NAME : public util::test_base {
 #define TEST_FAIL
     log.note("SYCL_LANGUAGE_VERSION not present");
 #else
+    static_assert(SYCL_LANGUAGE_VERSION, "202012L");
```

Comment:
Shouldn't this be something like:

```
static_assert(SYCL_LANGUAGE_VERSION == 202012L)
```

Followed by a check that the type is really `long`.

Reply:
> Followed by a check that the type is really `long`.

Something like:

```
static_assert(std::is_same_v<decltype(SYCL_LANGUAGE_VERSION), long>);
```